source/user-manual/ruleset/custom.rst

```diff
@@ -150,7 +170,11 @@ If we want to change something in the decoder file ``0310-ssh_decoders.xml``, we
       </ruleset>
 
 
-3. Perform the changes in the file ``/var/ossec/etc/decoders/0310-ssh_decoders.xml``.
+#. Perform the changes in the file ``/var/ossec/etc/decoders/0310-ssh_decoders.xml``.
+
+#. Restart the Wazuh manager so the changes take effect:
+
+   .. include:: /_templates/installations/manager/restart_wazuh_manager.rst
 
-    .. warning::
-        Note that at this point, if updates to the public Wazuh Ruleset include changes to 0310-ssh_decoders.xml, they will not apply to you since you are no longer loading that decoder file from the standard location that gets updates.  At some point, you may have to manually migrate your customized material from 0310-ssh_decoders.xml to a newer copy of that file.  Consider internally documenting your changes in 0310-ssh_decoders.xml so that they are easy to find if they have to be migrated later.
+   .. warning::
+      Note that at this point, if updates to the public Wazuh ruleset include changes to ``0310-ssh_decoders.xml``, they will not apply to you since you are no longer loading that decoder file from the standard location that gets updates.  At some point, you may have to manually migrate your customized material from ``0310-ssh_decoders.xml`` to a newer copy of that file.  Consider internally documenting your changes in ``0310-ssh_decoders.xml`` so that they are easy to find if they have to be migrated later.
```

Comment:
```suggestion
      Note that, since you are excluding the original decoder, you aren't benefiting from any updates it might get. You're no longer loading that decoder file from the standard location that gets the update.  Thus, you need to repeat your customization using the newer default copy of the file in this case.  Consider documenting your changes so that they are easy to find if you need to migrate them later.
```